include/modem/lte_lc.h

```diff
@@ -48,7 +48,39 @@ enum lte_lc_system_mode {
 	LTE_LC_SYSTEM_MODE_NBIOT,
 	LTE_LC_SYSTEM_MODE_GPS,
 	LTE_LC_SYSTEM_MODE_LTEM_GPS,
-	LTE_LC_SYSTEM_MODE_NBIOT_GPS
+	LTE_LC_SYSTEM_MODE_NBIOT_GPS,
+	LTE_LC_SYSTEM_MODE_LTEM_NBIOT,
+	LTE_LC_SYSTEM_MODE_LTEM_NBIOT_GPS,
+};
+
+/** LTE mode preference. If more than one LTE system mode is enabled, the modem
+ *  can select the mode that best meets the criteria set by this configuration.
+ *
+ *  Note that there's a distinction between preferred and prioritized mode.
+ */
+enum lte_lc_system_mode_preference {
+	/* No LTE preference, automatically selected by the modem. */
+	LTE_LC_SYSTEM_MODE_PREFER_AUTO = 0,
+	/* LTE-M is preferred over PLMN selection. If LTE-M is available in
```

Comment:
I'm not sure I see the difference between the original text and the suggested text. I think they both state the same? If LTE-M is not available in home network (where it will be searched for first), the modem will attempt to use LTE-M in a different network

Reply:
I just thought it was a little bit wordy. That we could convey the same message but with less text.  But as far as the message goes I think it is clear as day. No biggie.

Reply:
Oh I see. Updated the enum documentation to be (hopefully) more concise